tox.ini

```diff
@@ -1,33 +1,26 @@
 [tox]
 envlist =
-    {py3.6,py3.7,py3.8,py3.9}-django22
-    {py3.6,py3.7,py3.8,py3.9,py3.10}-django32
-    {py3.8,py3.9,py3.10}-django40
-    {py3.8,py3.9,py3.10,py3.11}-django41
     {py3.8,py3.9,py3.10,py3.11,3.12}-django42
     {py3.10,py3.11,3.12}-django50
-    py3.9-flake8
+    {py3.10,py3.11,3.12}-django51
+    {py3.10,py3.11,3.12}-django52
```

Comment:
This env doesn't appear to be used, but it doesn't break CI so I'm fine with it - we can tweak after it's merged, if need be

Reply:
My mistake, I think we can confirm Django 5.2 support as well, #38.

Reply:
I'm actually already running a 5.2rc1 [in a draft](https://github.com/janbrasna/django-allow-cidr/actions/runs/13996414127/job/39192583999) so I'll just follow up with it and tidy up any version support that's left, so don't worry about these…